AnkiDroid/build.gradle

```
@@ -214,6 +215,7 @@ dependencies {
         resolutionStrategy {
             force 'org.jetbrains:annotations:20.1.0'
         }
+        coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.9'
     }
 
     compileOnly 'org.jetbrains:annotations:20.1.0'
```

Sorry I did not notice this earlier, but you've added it in the wrong spot I think.
We don't want it in the `configurations.all` block - that's a special hack I put in for a specific reason as documented in the comment.

As documented for Android Studio https://developer.android.com/studio/releases#4-0-0-desugar you just add this to the regular dependencies list

```suggestion
    }

    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.9'
    compileOnly 'org.jetbrains:annotations:20.1.0'
```